libraries/omi/meta/summary.md

```diff
@@ -0,0 +1,7 @@
+<h4 id="omi-handling-data">Handling data</h4>
+
+Omi passes all data to Custom Elements as attributes. 
```

Comment:
This doesn't seem right, otherwise the advanced property tests won't pass, will they?

Specifically the test: advanced support -> attributes and properties -> will pass object data as a property

It looks like Omi behaves similarly to Preact, in that it will set the property if it exists on the element, and otherwise it will set an attribute.

Reply:
done